Why Elm.md

```diff
@@ -27,7 +27,7 @@ You may wonder how Elm developers can survive with all these restrictions. The s
 
 Developing with Elm often feels like an ongoing conversation with the compiler, gently pushing you toward writing better code. Often, you don't even need to look at your application in the browser while implementing a feature — you can trust the compiler to tell you if you are going in the right direction.
 
-You may be asking yourself why you should choose to learn a language that's so different from other tools in the ecosystem. **We'd argue that the experience of working in an alien environment can be productive, even if it's just for a little while.**  You are probably not used to dealing with the constraints of purity, immutability, and static typing, which together make up the basic tenets of **functional programming**. These constraints will push you to solve problems in creative ways, and hopefully you will learn strategies that you can employ in other languages.
+You may be asking yourself why you should choose to learn a language that's so different from other tools in the ecosystem. **We'd argue that the experience of working in an alien environment can make you more productive in your home environment, even if the experience is just for a little while.**  You are probably not used to dealing with the constraints of purity, immutability, and static typing, which together make up the basic tenets of **functional programming**. These constraints will push you to solve problems in creative ways, and hopefully you will learn strategies that you can employ in other languages.
```

Comment:
hm, the reference to your "home environment" is a little awkward, and it be may unclear — home could mean "not at work" in this context. What if we kept the original text but changed `productive` to `fruitful`, to make it clear that the gains are unrelated to a particular environment?